chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -105,7 +105,7 @@ const MAX_MESSAGES_TOTAL_SIZE: usize = usize::MAX;
 macro_rules! unwrap_or_error(($obj: expr, $error: expr) => (match $obj {
     Ok(result) => result,
     Err(err) => {
-        error!(target: "network", "{}: {}", $error, err);
+        error!(target: "network", message = "", error = ?$error, err = ?err);
```

Comment:
where are we changing the implementation to use tracing ? (or does it happen automatically?)

Reply:
@mm-near We are already using `tracing` in `near-network`.